Sources/AloeStackView/AloeStackView.swift

```diff
@@ -189,6 +189,7 @@ open class AloeStackView: UIScrollView {
     guard let cell = row.superview as? StackViewCell, cell.isHidden != isHidden else { return }
 
     if animated {
+      cell.layoutIfNeeded()
       UIView.animate(withDuration: 0.3) {
         cell.isHidden = isHidden
         cell.layoutIfNeeded()
```

Comment:
Is this second layout call needed now?

Reply:
Yes, it is the second call, within the animation block, that actually animates the constraint change (via isHidden).

That said, I spoke too soon and this did not actually fix my problem.  What is happening is multiple calls to setRowHidden(true) are "accumulating" and setRowHidden(false) does not take effect until it's called enough times to "balance" all the hides.  There is a unit test in the project for this scenario - and it passes - but that is exactly what I'm seeing.

I've presently wrapped the call to setRowHidden() in isRowHidden() to execute only when there's a change but that's, well, gross.  I'm perplexed.